integration/tekton/tasks/ike-create/ike-create.yaml

```diff
@@ -37,4 +38,9 @@ spec:
       image: released-image
       script: |
         STATE=`ike create --session $(params.session) --route $(params.route) --deployment $(params.target) --namespace $(params.namespace) --image $(params.image) --json`
```

Comment:
- [ ] If route not set (default empty) we shouldn't pass `--route` param at all.